### ISSUE TITLE
training net with landmark annotation

### ISSUE THREAD

**Issue by Borda** [voxelmorph/voxelmorph#41]
Hello, I am very curious about your method, looks very promising. In the readme you mentioned that is unsupervised - _"Unsupervised Learning with CNNs for Image Registration"_ and later there is a training phase (after setup), so I become confused... If training with annotation is needed, would it work with a landmark annotation such as [ANHIR](https://anhir.grand-challenge.org)?

**Comment by adalca** [comment]
Hi @Borda , 

The method is unsupervised, in that no annotations are needed. There *is* a training phase, and a loss function, but this training and loss do not use any annotations, rather they measure the 'goodness' of the deformation produced by the network by checking if warping the first image with this deformation gives you the second image (plus some regularization).

This is described in more details in our papers, see for example: https://arxiv.org/abs/1809.05231

If you happen to have landmarks during training, you can optionally use them (the paper above describes the concept behind this as well), but you don't *need* annotations to train.

**Comment by Borda** [comment]
@adalca thank you for your fast reply. May I ask if you would consider your participation in the ANHIR challenge?

**Comment by adalca** [comment]
We've certainly thought of it, and I think voxelmorph would do decently, but honestly we've just not had the time. If the challenge is completed, how could we participate?

**Comment by Borda** [comment]
if you succeed to have a submission in some reasonable time (next couple of weeks), we would include your method in the overview journal paper which we are preparing. I am happy to continue our discussion via [mail](http://cmp.felk.cvut.cz/~borovji3)
